tests/Models/TokenSigningTraitTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @author Chris Chabilall <chris.c@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+namespace VanillaTests\Models;
+
+use VanillaTests\SharedBootstrapTestCase;
+use VanillaTests\SiteTestTrait;
+use VanillaTests\Fixtures\TokenTestingModel;
+
+/**
+ * Test the {@link TokenSigningTrait}.
+ * Used to test the token generation and signing utility methods.
+ */
+class TokenSigningTraitTest extends SharedBootstrapTestCase {
+    use SiteTestTrait;
+
+    /**
+     * Tests random token generation and signing.
+     */
+    public function testVerifyRandomTokenSignature() {
+        $model = new TokenTestingModel();
+        $token = $model->signToken($model->randomToken(), '2 months');
+        $this->assertEquals(true, $model->verifyTokenSignature($token, true));
+    }
+
+    /**
+     * An expired token shouldn't verify.
+     *
+     * @expectedException \Exception
+     * @expectedExceptionMessage Your nonce has expired.
+     */
+    public function testExpiryDate() {
+        $model = new TokenTestingModel();
+        $token = $model->signToken($model->randomToken(), 'last year');
+        $this->assertEquals(false, $model->verifyTokenSignature($token, true));
```

Comment:
If we're expecting exceptions, it doesn't seem like these assertions are actually doing anything. Do they need to be there or can we remove them?

Reply:
the previous tests on the accesstokenmodel had assertions, I left them.  I don't think they're doing much, get rid of them?

Reply:
So long as PHPUnit doesn't complain about it (sometimes it doesn't like assertion-less tests), getting rid of the assertions in tests where we're expecting exceptions should be fine.